cellarium/cas/client.py

```diff
@@ -565,7 +574,7 @@ def annotate_anndata_file(
             keyword, which refers to the default selected model in the Cellarium backend. |br|
             `Default:` ``"default"``
         :param count_matrix_input:  Where to obtain a feature expression count matrix from. |br|
-            `Allowed Values: Choices from enum :class:`cellarium.cas.constants.CountMatrixInput` |br|
+            `Allowed Values: Choices from enum :class:`constants.CountMatrixInput` |br|
```

Comment:
The other one was because that's an rST file so there's no python import context to to the class resolution.  Though I'm looking. now and it doesn't seem to be working.  Lemme look again

Reply:
Oh wait, stray quote was messing stuff up...

Reply:
Ok, I'll shorten these to use the wildcard notation and that should be a lot more concise (e.g. it'll look like `~.CountMatrixInput` (or `~.constants.CountMatrixInput` if we ever need to be more specific)